pkg/operator/cassandra/controller/cluster.go

```diff
@@ -0,0 +1,69 @@
+package controller
+
+import (
+	"fmt"
+	cassandrav1alpha1 "github.com/rook/rook/pkg/apis/cassandra.rook.io/v1alpha1"
+	"github.com/rook/rook/pkg/operator/cassandra/controller/util"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+)
+
+const (
+	StatefulSetName = "%s-%s-%s"
```

Comment:
comment on the format of this string? (what are the 3 expected params?)

Reply:
Naming of StatefulSets was moved into a function.